go/counter.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"github.com/concord/concord-go"
+	"log"
+	"strconv"
+	"time"
+)
+
+// Counter holds computation state
+type Counter struct {
+	stats map[string]int64
+}
+
+// Metadata is called by Concord to define input/output streams for computation
+// This computation only reads data from "words" stream
+func (c *Counter) Metadata() *concord.Metadata {
+	return &concord.Metadata{
+		Name:   "counter",
+		Inputs: []*concord.Stream{concord.NewDefaultStream("words")},
+	}
+}
+
+// Initialization of computation. This method is called by Concord
+func (c *Counter) Init(ctx *concord.Context) error {
+	log.Println("[INFO] initialize word's counter")
+	c.stats = make(map[string]int64)
+
+	// schedule next ProcessTimer call
+	ctx.SetTimer(time.Now(), "")
```

Comment:
Can you make a fix on the go-api so that the `key:string` comes first then the `time:int64` comes second?  - This just keeps it consistent with the rest of the clients.